tests/utils/stl/test/tests.py

```diff
@@ -36,9 +36,11 @@ def __init__(self, suite, path_in_suite, lit_config, test_config,
 
         # TRANSITION: These configurations should be enabled in the future.

```

Comment:
Pre-existing: Don't we have bug numbers we could put in this comment?

Reply:
We do now!